routes/router.js

```diff
@@ -103,34 +101,24 @@ module.exports = function(app, passport, path) {
 		req.session.destroy(function (err) {
 			res.status(302).redirect('/');
 		});
-		// res.sendFile(path.resolve(__dirname + '/../public/views/login.html'));
 	})
 
-	app.put('/location', (req, res) => {
-		console.log(req.body, 'body here');
+	app.put('/location/:id', (req, res) => {
 		User
 		.findOne(
-			{_id: req.body.user._id}
+			{_id: req.params.id}
 		).exec(function (err, data) {
-			console.log(data, 'data here')
+			console.log(arguments)
+			if(err) {
+				return res.status(500).send(err);
```

Comment:
If that's the case, this is okay.  I haven't added any 2-space rules or anything.